data/entities.json

```diff
@@ -750,4 +750,10 @@
     "categories": ["marketing"],
     "domains": ["cdn.listrakbi.com", "s1.listrakbi.com"]
   }
+  {
+    "name": "Ontame",
+    "homepage": "https://www.ontame.io",
+    "categories": ["customer-success"],
```

Comment:
Their website seems like they provide tracking for measuring hiring pipelines. Is this accurate? If so, I might put this into the "analytics" bucket.

Reply:
I don't know, they all look the same to me :D I've changed the category.